pages/portals/coldfront.md

```diff
@@ -41,6 +41,9 @@ project has been created take a moment to add any publications, grants, or
 research output which helps us keep track of the scientific impact of our
 center. You can also [add users](#add-or-remove-users) to your project.
 
+!!! Note "New Faculty Must Request PI Status"  
```

Comment:
We also have this same exact thing [here](https://docs.ccr.buffalo.edu/en/latest/getting-access/#enable-two-factor-authentication) but worded differently. I'm not sure it's worth repeating again here but if you think it is, let's use the exact same wording?

Reply:
I did not see this!  I was wondering why we didn't have it somewhere so I added it.  I've removed this though as I agree we don't need to repeat it.